pkg/vsphere/disk/disk_manager.go

```diff
@@ -421,3 +423,51 @@ func (m *Manager) devicePathByURI(op trace.Operation, datastoreURI *object.Datas
 
 	return sysPath, nil
 }
+
+// AttachAndMount creates and attaches a vmdk as a non-persistent disk, mounts it, and returns the mount path.
+func (m *Manager) AttachAndMount(op trace.Operation, datastoreURI *object.DatastorePath) (string, error) {
+	config := NewNonPersistentDisk(datastoreURI)
+	d, err := m.CreateAndAttach(op, config)
```

Comment:
@hickeng do you mean creating a new non-persistent delta disk with the actual target disk being the parent vs. attaching an existing disk? If that's what you mean, wouldn't that result in more disks being attached (albeit non-persistent children of the targets you are diffing between)? Versus just `Attach`ing an existing disk (which would just bump the reference count if it was already attached)? In a concurrent diff scenario, the former sounds like you could be creating many non-persistent delta disks where the latter seems like you would be simply reusing already attached disks.